packages/mds-agency/utils.ts

```diff
@@ -499,7 +499,8 @@ const normalizeTelemetry = (telemetry: TelemetryResult) => {
   return telemetry
 }
 
-export async function readPayload(store: typeof cache | typeof db, device_id: UUID): Promise<VehiclePayload> {
+export async function readPayload(cached: boolean, device_id: UUID): Promise<VehiclePayload> {
```

Comment:
Acceptance criteria for the ticket said that for reading a single device should _always_ use the cache for reading event data, while it should use the db for reading back device data still. Not one or the other (which is how it was previously, and is still implemented) by the `cached` property.